tox.ini

```diff
@@ -19,6 +19,15 @@ passenv = DATEUTIL_MAY_CHANGE_TZ TOXENV CI TRAVIS TRAVIS_* APPVEYOR APPVEYOR_* C
 commands = python -m pytest {posargs: "{toxinidir}/dateutil/test" --cov-config="{toxinidir}/tox.ini" --cov=dateutil}
 deps = -rrequirements-dev.txt
 
+[testenv:py33]
```

Comment:
This line creates a new py33-only `testenv` just so that I can use different requirements/constraints files, everything else is the same.

@gaborbernat Do you think there's a more effective way to do this? I tried using environment selectors on the requirements.txt files, but tox didn't understand it.

In addition to the DRY violation, the big downside to doing this is that apparently tox -e py won't work properly on Python 3.3, because it won't automatically select the py33 environment.